docs/detections/api/rules/signals-api-overview.asciidoc

```diff
@@ -245,3 +250,107 @@ Example response:
   "failures": []
 }
 --------------------------------------------------
+
+
+==== Apply alert tags
+
+Add and remove alerts tags on alerts.
+
+===== Request URL
+
+`POST <kibana host>:<port>/api/detection_engine/signals/tags`
+
+===== Request body
+
+A JSON object with the `tags` and `_id` fields:
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`tags` |String[] a|Array of alert tags. Alert tags are the words and phrases that help categorize alerts. 
+
+Properties of the `tags` objects:
+
+* tags_to_add: (Required, string[]) Array of tags you want to add.
+* tags_to_remove: (Required, string[]) Array of tags you want to remove.
+
+NOTE: You cannot add and remove the same alert tag. 
+
+|No 
+
+|`ids` |String[] |Array of alert IDs.|Yes
+
+|==============================================
+
+====== Example requests
+
+Adds tags to alerts:
+
+[source,console]
+--------------------------------------------------
+POST api/detection_engine/signals/tags
+{
+  "tags": {
+    "tags_to_add": ["False Positive"],
+    "tags_to_remove": ["Further Investigation Required"]
+  },
+  "ids": [
+    "694156bbe6a487e06d049bd6019bd49fec4172cfb33f5d81c3b4a977f0026fba",
+    "f4d1c62c4e8946c835cb497329127803c09b955de49a8fa186be3899522667b0"
+  ]
+}
+--------------------------------------------------
+
+===== Response code
+
+`200`::
+    Indicates a successful call.
+
+====== Response payload
+
+A JSON object containing tagged alerts.
```

Comment:
Does the response return: 
only alerts that have had tags added to them (tagged alerts) 
or
alerts that have had tags either added or removed (updated alerts)?

If it's the latter, then I would consider changing "containing tagged alerts" to "containing updated alerts" or something to that effect, to indicate that alerts that aren't tagged anymore will also be returned.

Reply:
Great suggestion! I believe it's the latter (alerts that have had tags either added or removed) but I'll let @dplumlee confirm.

Reply:
Yeah it's alerts that were updated in general not just added

Reply:
Thanks for confirming, @dplumlee!

@natasha-moore-elastic how does this description look:

```suggestion
A JSON object containing alerts with newly added or removed alert tags. 
```

Reply:
Sounds good, I like that it's more specific!